### ISSUE TITLE
Misleading telephone number as link guidance

### ISSUE THREAD

**Issue by tobias-edwards** [alphagov/govuk-design-system#2022]
https://github.com/alphagov/govuk-design-system/blob/aaae72cf605a7f00e0afee025e2e109943935f49/src/patterns/telephone-numbers/index.md.njk#L79

Desktop devices are able to make telephone calls, and in terms of making telephone numbers fully accessible, I would have thought telephone numbers should be displayed as links?

**Comment by EoinShaughnessy** [comment]
@tobias-edwards Thanks for raising this issue!

I'll update this content to correct the inaccurate reference to desktop devices.

I've talked to some designers, developers and accessibility specialists about the accessibility angle. Several of them cautioned against styling phone numbers as links. In some cases, clicking on a linked number could trigger an app-installation process. For example, Skype. The feeling was, this behaviour may be an unwelcome turn in the user's journey.

However, I also got feedback about organisations that style phone numbers as links, and allow users to decide for themselves how to interact.

So, there are different schools of thought. I've shared your question with our GDS accessibility community, and we're going to discuss it further at our next catch-up.

**Comment by tobias-edwards** [comment]
Taking the Skype example, for me it would be unwelcome but for someone who has limited accessibility and relies on Skype to make phone calls it's very helpful. Certainly feels like a catch-22.

Within the same passage, there's an argument not to display phone numbers as links on mobile browsers:
https://github.com/alphagov/govuk-design-system/blob/aaae72cf605a7f00e0afee025e2e109943935f49/src/patterns/telephone-numbers/index.md.njk#L81

But visiting https://www.gov.uk/government/publications/contacting-an-official-receiver/contact-information#Contacting-Official-Receivers-offices on my phone does not display these phone numbes as links:
- Device: Pixel 3a XL
- OS: Android 12
- Browsers: Chrome 98.0.4758.101 and Firefox 97.1.0

I look forward to hearing what comes out of your catch-up.

**Comment by EoinShaughnessy** [comment]
Thanks @tobias-edwards!

We're also correcting that line so that it reads, instead,

> It might also not be necessary - some modern mobile browsers automatically detect telephone numbers and display them as links anyway.

I'll also copy in my accessibility specialist teammate @davidc-gds so he has visibility of this convo.

**Comment by davidc-gds** [comment]
Hey @tobias-edwards!

I'll pop in just to say my plan is to add this to the agenda for our next GDS accessibility community meeting, which is on March 8th. That should be a good forum to discuss the different schools of thought around phone number styling that Eoin mentioned.

I doubt we'll have any further update before March 8th, beyond the changes Eoin has already mentioned, but stay tuned!

**Comment by tobias-edwards** [comment]
Was there any follow-up to this? Is the drafted PR https://github.com/alphagov/govspeak/pull/207 the resolution?

**Comment by EoinShaughnessy** [comment]
Hi @tobias-edwards, sorry for the late reply - I've just come back after being away.

The drafted PR is for govspeak, which is a different service to the Design System. Anything done in that repo has no effect on the Design System site, and we're not the team that looks after it.

We raised the issue at the accessibility community meetup. People felt it would be good to revisit our original reasoning for not styling phone numbers as links, to find out whether it still seems valid. How soon that work could be prioritised, however, is hard to say.

If some other users agreed about styling phone numbers as links, that would help make this issue a higher priority. The telephone numbers pattern has its own [discussion page](https://github.com/alphagov/govuk-design-system-backlog/issues/101) - that might be a good place to continue this conversation, as more people will see it than in this issue.

**Comment by tobias-edwards** [comment]
Yesterday I spoke with an accessibility lead at my company who shared his views on the issue. I've presented his arguments below:

# Telephone should be links

https://github.com/alphagov/govuk-design-system/blob/0d75a84be5b76c12a6747861ba2af58f29c9fae3/src/patterns/telephone-numbers/index.md.njk#L81

We should not rely on a user agent to automatically identify unlinked phone numbers as telephone numbers. This behaviour is not part of any web standard for a browser to do this, and phone number formats vary greatly from region to region.

Some screen reader users will struggle to copy-paste a telephone number into their VoIP and it's simply much easier to have it as a link.

https://github.com/alphagov/govuk-design-system/blob/0d75a84be5b76c12a6747861ba2af58f29c9fae3/src/patterns/telephone-numbers/index.md.njk#L85

I can sympathise with this argument, but once a user knows a telephone number presented as a link will try to open up their VoIP client, if they don't want this behaviour in the future, then they can just not click it in the future.

https://github.com/alphagov/govuk-design-system/blob/0d75a84be5b76c12a6747861ba2af58f29c9fae3/src/patterns/telephone-numbers/index.md.njk#L79
https://github.com/alphagov/govuk-design-system/blob/0d75a84be5b76c12a6747861ba2af58f29c9fae3/src/patterns/telephone-numbers/index.md.njk#L83

Most devices are capable of making phone calls, including desktop devices via Skype, Teams, etc. I would struggle to find a device that is incapable of making phone calls.

# Do not use `aria-label`

i.e. `<a href="tel:7035551212" aria-label="7 0 3. 5 5 5. 1 2 1 2.">(703) 555-1212</a>`

This issue is somewhat separate to this thread but still interesting. We should not use `aria-label` on telephone numbers to 'improve' their pronounciation by screen readers because:

- Aria labels should not be used to correct pronouciation of text
- Screen reader users know how their screen readers work over time
- An aria label may prevent users from navigating correctly through the document character by character to read the phone number themselves
- The aria label may not match the visible phone number (by mistake)

**Comment by EoinShaughnessy** [comment]
Thanks @tobias-edwards! That's really valuable, detailed feedback. I'll relay what you've shared to the rest of the team. I've also shared this feedback within the [discussion page for the telephone numbers pattern](https://github.com/alphagov/govuk-design-system-backlog/issues/101#issuecomment-1098217527). That might help get community support for any potential changes.
